parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -85,6 +85,7 @@ public static WriteBuilder write(OutputFile file) {
     private Map<String, String> config = Maps.newLinkedHashMap();
     private Function<MessageType, ParquetValueWriter<?>> createWriterFunc = null;
     private MetricsConfig metricsConfig = MetricsConfig.getDefault();
+    private ParquetFileWriter.Mode writeMode = ParquetFileWriter.Mode.OVERWRITE;
```

Comment:
Can this default to non-overwrite mode?

Reply:
Now all readers default to overwrite false.